components/style/properties/longhand/font.mako.rs

```diff
@@ -27,7 +27,7 @@
 
         #[derive(Debug, PartialEq, Eq, Clone, Hash, HeapSizeOf, Deserialize, Serialize)]
         pub enum FontFamily {
-            FamilyName(Atom),
+            FamilyName(String),
```

Comment:
Gecko atoms don't have a utf-8 representation, so it's impossible to support `FontFamily::name()` with it.